mathml/relations/css-styling/lengths-2.html

```diff
@@ -57,14 +56,12 @@
       assert_equals(getBox("spacePercentage").width, 60, "%");
       assert_equals(getBox("spacePt").width, 96, "pt");
       assert_equals(getBox("spacePx").width, 123, "px");
-      assert_equals(getBox("spaceNone").width, 150, "Unitless");
     }, "Trimming of space");
 
     test(function() {
       assert_approx_equals(getBox("n0").width, 0, epsilon, "n0");
       assert_approx_equals(getBox("n1").width, 90, epsilon, "n1");
       assert_approx_equals(getBox("n2").width, 8, epsilon, "n2");
-      assert_approx_equals(getBox("n3").width, 70, epsilon, "n3");
```

Comment:
Now we leave a gap, I think it is less confusing if we have a range n0 till n5 with no gaps.

Reply:
Ah, never mind, did not scroll down enough.